styles/foobar-ui.css

```diff
@@ -66,6 +66,9 @@
   --space-xl: 2.5rem;
   --space-xxl: 4rem;
 
+  /* Border Radii */
+  --border-radius-full: 50%;
+
```

Comment:
This code also exists in `variables.css`. Is that a requirement?

Reply:
my foobar-ui.css is a generated stylesheet. variables.css is where the code is originally written